lms/services/dashboard.py

```diff
@@ -88,6 +90,26 @@ def delete_dashboard_admin(self, dashboard_admin_id: int) -> None:
         """Delete an existing dashboard admin."""
         self._db.query(DashboardAdmin).filter_by(id=dashboard_admin_id).delete()
 
+    def get_request_admin_organizations(self, request) -> list[Organization]:
+        """Get the organization the current user is an admin in."""
+        if request.has_permission(Permissions.STAFF) and (
+            request_public_id := request.params.get("public_id")
+        ):
+            # We special case permissions/filtering for staff members
```

Comment:
Is there something missing between `We` and `special`? Or did you mean something different than `We`?

Reply:
I was trying to unsuccessfully use ["special case"](https://en.wiktionary.org/wiki/special_case#English) a verb here.

Reworded it to be more clear.

Reply:
Oh, I see. Actually now I get the meaning 😅